### PR TITLE
add sample e2e tests

### PR DESCRIPTION
**Pull request by timvahlbrock**
### Description

Created some sample E2E Tests.

## Checklist

- [x] My code follows the style guidelines
- [x] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [x] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [x] Backend
- [x] Github
- [x] Root

## If tests are required did you make them?

- [ ] No tests required
- [x] Yes
- [ ] No

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -91,54 +90,6 @@ jobs:
...
 
-
-    frontend_container_arm64:
-      name: Build arm64 Frontend Container
-      needs: frontend_lint_test_build
-      runs-on: ubuntu-latest
-      permissions:
-        contents: read
-        packages: write
-      steps:
```
Comment (Paulomart) [inline .github/workflows/ci.yml:102]: This seems to remove the arm builds for some reason?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -342,10 +245,34 @@ jobs:
           image: oeffis/backend:${{ github.sha }}-untested-amd64
           ports:
             - 3000:3000
+          env:
+            VRR_BASE_URL: http://mocks-server:3100
+          # options: >-
+          #   --health-cmd "curl -L --fail http://localhost:3000"
+          #   --health-interval 10s
+          #   --health-timeout 5s
+          #   --health-retries 5
```
Comment (Paulomart) [inline .github/workflows/ci.yml:254]: here are still comments?

**Review by Paulomart — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (timvahlbrock) [reply .github/workflows/ci.yml:102]: No, the arm builds are just fully moved to "after E2E has passed".

File: .github/workflows/ci.yml (hunk 2 above)
Comment (timvahlbrock) [reply .github/workflows/ci.yml:254]: Yes. I originally planned to enable health checks again at a later point, but skipped it for now. I will remove the comments.

Comment (timvahlbrock) [reply .github/workflows/ci.yml]: Done.

**Review by Paulomart — APPROVED** [review]

**Pull request merged**